frontend/src/views/web/challenge/edit-challenge/edit-challenge-submission-guidelines.html

```diff
@@ -1,22 +1,22 @@
-<section class="text-center pd-20 width-500">
+<section class="text-center pd-20 width-700">
```

Comment:
Also, where is this class defined? I think, currently we have only 'width-500' class presdent. You have to create a new one for 'width-700'. @S2606

Reply:
@aka-jain ok thank you!!